src/profiler/components/VerticalTable.tsx

```diff
@@ -3,7 +3,7 @@ import Table from 'antd/lib/table'
 
 import ChartCellTitle from './ChartCellTitle'
 import { ChartAndTable, hygratePmfPlotData } from '../chart'
```

Comment:
![Screenshot 2021-11-25 at 15 20 54](https://user-images.githubusercontent.com/9988008/143449125-3f8acb33-f6bc-429e-a504-0c5b7f666807.png)

please, rebase from the `main` branch